lib/generated_definitions/au.rb

```diff
@@ -48,7 +48,8 @@ def self.holidays_by_month
       8 => [{:wday => 3, :week => -3, :name => "Ekka", :regions => [:au_qld_brisbane]},
             {:wday => 1, :week => 1, :name => "Picnic Day", :regions => [:au_nt]}],
       9 => [{:wday => 1, :week => -1,  :year_ranges => [{:before => 2022}],:name => "Queen's Birthday", :regions => [:au_wa]},
-            {:wday => 1, :week => -1,  :year_ranges => [{:after => 2023}],:name => "King's Birthday", :regions => [:au_wa]},
+            {:wday => 1, :week => -1,  :year_ranges => [{:after => 2025}],:name => "King's Birthday", :regions => [:au_wa]},
+            {:mday => 23,  :year_ranges => [{:limited => 2024}],:name => "King's Birthday", :regions => [:au_wa]},
```

Comment:
Was the 'after' inclusive as well? If so we probably want to leave 2023 in here? I guess that means it needs a manually addition as well..